### PR TITLE
Dp 23317 sidewide alerts styles

### PR DESCRIPTION
**Pull request by clairesunstudio**
Link to MF https://github.com/massgov/mayflower/pull/1547

we are going to do a quick-win version to reduce the vertical height of site-wide alerts -->

- Keep the same icon for now for both types of alerts
- Change the header prefix “Emergency Alerts” → “Alerts”, “Informational Alerts” → “Notices”
- Keep the header prefix visible on all screensizes

See details on MF PR https://github.com/massgov/mayflower/pull/1547

### CODE REVIEW

**Comment by clairesunstudio** [comment]
UAT --> https://pr1131-9flgqvpap3e3ypjm5vcs2wjnwfuz9pte.tugboat.qa/alerts/coronavirus-update

**Comment by clairesunstudio** [comment]
- [x] Emergency Alerts --> Alerts
Screen Shot 2021-11-04 at 12 52 51 PM

- [x] Informational Alerts --> Notices change isn't reflected. Can someone point me to the place to update that? @tom-fleming @joeg8612 
Screen Shot 2021-11-04 at 12 53 25 PM

**Comment by clairesunstudio** [comment]
The issue with changing Informational Alerts --> Notices is resolved. (We are hard coding the alerts header --> 90f0841)
Screen Shot 2021-11-05 at 10 36 08 AM

File: changelogs/DP-23317.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# You may add more than 1 description & issue for each type using the following format:
+# Changed:
+#  - description: Automating the release branch.
+#    issue: DP-10166
+#  - description: Second change item that needs a description.
+#    issue: DP-19875
+#  - description: Third change item that needs a description along with an issue.
+#    issue: DP-19843
+#
+Type:
```
Comment (Javi-er) [inline changelogs/DP-23317.yml:39]:
```suggestion
Changed:
```

**Comment by clairesunstudio** [comment]
replaced by https://github.com/massgov/openmass/pull/1155

**Pull request closed without merging**